app/app.go

```diff
@@ -410,10 +430,15 @@ func NewApp(
 	app.SetBeginBlocker(app.BeginBlocker)
 
 	app.SetAnteHandler(
-		auth.NewAnteHandler(
-			app.keeper.acct,
-			app.keeper.supply,
-			auth.DefaultSigVerificationGasConsumer,
+		chainAnteChains(
+			auth.NewAnteHandler(
+				app.keeper.acct,
+				app.keeper.supply,
+				auth.DefaultSigVerificationGasConsumer,
+			),
+			sdk.ChainAnteDecorators(
+				dante.NewDecorator(app.keeper.deployment),
```

Comment:
:metal: :fire: `dante.Inferno()` :fire: :metal:

Reply:
:boom: tbh I wanted to name it this way initially